readme.md

```diff
@@ -1,4 +1,4 @@
-<div align="center">
+Œ<div align="center">
```

Comment:
I have no idea what happened in this commit. Looks like the GitHub web UI completely borked up the commit.